core/ops.rs

```diff
@@ -177,9 +158,9 @@ impl OpCtx {
     // version and require a slightly different set of arguments (for example, it may need the fastcall
     // callback information to get the `OpCtx`).
     let fast_fn = if metrics_fn.is_some() {
-      &decl.fast_fn
-    } else {
       &decl.fast_fn_with_metrics
+    } else {
+      &decl.fast_fn
```

Comment:
Ooops, a previous mistake?

Reply:
Yeah 😓  We never actually enabled metrics in fast ops. I added the fast ops to the benchmark so it'll have some coverage.